addons/website/static/tests/tours/focus_blur_snippets.js

```diff
@@ -0,0 +1,79 @@
+odoo.define(function (require) {
+'use strict';
+
+const ajax = require('web.ajax');
+const tour = require('web_tour.tour');
+
+const blockIDToData = {
+    parent: {
+        selector: '.s_focusblur',
+        name: 'section',
+        number: 0,
+    },
+    child1: {
+        selector: '.s_focusblur_child1',
+        name: 'first child',
+        number: 1,
+    },
+    child2: {
+        selector: '.s_focusblur_child2',
+        name: 'second child',
+        number: 2,
+    },
+};
+
+function clickAndCheck(blockID, expected) {
+    const blockData = blockIDToData[blockID] || {};
+
+    return [{
+        trigger: blockData.selector || '#wrapwrap',
```

Comment:
I'd definitely add a name here, will be a lot easier when debugging this test/tour (especially reading the logs to follow the steps).
Something like `Clicking on ${blockID}, expecting those events: ${expected}`

Edit: After writing this, I saw the content next line :grimacing: wasn't expecting it to not be first, can you add it as first key?